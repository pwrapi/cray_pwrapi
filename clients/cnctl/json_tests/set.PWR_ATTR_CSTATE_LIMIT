{
 "PWR_Function"          : "PWR_ObjAttrSetValues",
 "PWR_Attrs"             : [
                            {
                              "PWR_AttrName" : "PWR_ATTR_CSTATE_LIMIT",
                              "PWR_AttrValue" : "4"
                            }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
