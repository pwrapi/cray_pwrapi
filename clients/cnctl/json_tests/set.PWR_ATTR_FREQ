{
 "PWR_Function"          : "PWR_ObjAttrSetValues",
 "PWR_Attrs"             : [
                            {
                              "PWR_AttrName" : "PWR_ATTR_FREQ",
                              "PWR_AttrValue" : "1500000"
                            }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
