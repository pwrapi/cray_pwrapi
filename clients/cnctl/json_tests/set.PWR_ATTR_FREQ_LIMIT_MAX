{
 "PWR_Function"          : "PWR_ObjAttrSetValues",
 "PWR_Attrs"             : [
                            {
                              "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MAX",
                              "PWR_AttrValue" : "2301000"
                            }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
