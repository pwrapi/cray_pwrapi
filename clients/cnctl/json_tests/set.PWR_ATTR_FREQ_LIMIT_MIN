{
 "PWR_Function"          : "PWR_ObjAttrSetValues",
 "PWR_Attrs"             : [
                            {
                              "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MIN",
                              "PWR_AttrValue" : "1200000"
                            }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
