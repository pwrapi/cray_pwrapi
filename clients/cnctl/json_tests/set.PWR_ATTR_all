{
 "PWR_Function"          : "PWR_ObjAttrSetValues",
 "PWR_Attrs"             : [
                            {
                              "PWR_AttrName" : "PWR_ATTR_CSTATE_LIMIT",
                              "PWR_AttrValue" : "4"
                            },
                            {
                              "PWR_AttrName" : "PWR_ATTR_FREQ",
                              "PWR_AttrValue" : "1500000"
                            },
                            {
                              "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MAX",
                              "PWR_AttrValue" : "2301000"
                            },
                            {
                              "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MIN",
                              "PWR_AttrValue" : "1200000"
                            }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
