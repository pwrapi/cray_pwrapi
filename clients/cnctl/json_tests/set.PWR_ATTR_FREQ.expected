{
 "PWR_MajorVersion"      : 0,
 "PWR_MinorVersion"      : 1,
 "PWR_Attrs"             : [],
 "PWR_ErrorMessages"     : [ "PWR_ATTR_FREQ is read-only" ],
 "PWR_ReturnCode"        : -8,
 "PWR_Messages"          : null
}
